no-computer.md

```diff
@@ -10,41 +10,82 @@ good way to meet their needs.
 _What need is met by your idea? Who are you helping? What is that person's
 experience like now? What would their experience be like if you could help 
 them?_
+My non-computer DSL idea is actually an abstract game I am in the process of 
+creating for my analog game design class. The game is called ["Circle of Paranoia"](https://docs.google.com/document/d/1WssK6n2gQf6TpReBtXNDcGkmX7z2NeDntuUo4jmU3aI/edit?usp=sharing), 
+and the game meets the need of people who are bored and want to play an amusing 
+game that requires some spatial thinking! It is a very accessible game that young
+kids could play using luck, but for adults, it could be a very competitive strategy
+game. Many abstract strategy games require a lot of time, but this game is pretty 
+short and fun to play in your spare time. 
 
 
 ### Why a language?
 _Why is a DSL appropriate for your user(s)? How does it address the need?_
+In this game, I would consider the different colored pieces as the vocabulary
+of the DSL, and the different positions on the board as the governing syntax. 
+The DSL is necessary for the players to communicate their moves and to become
+closer to reaching an end state of the game. They use the language by swapping 
+one piece with a piece in a different position on the circle or in the center of
+the circle. The "code" is modified during each player's turn when they must swap
+a piece. 
 
 
 ### Why you?
 _What excites you about this idea? How did you come up with it?_
+This idea excites me because I'm looking forward to play testing it with others
+and seeing their reactions. It's nice to make something that brings out a 
+competitive spirit in people and allows them to have some fun. I came up with it
+while brainstorming ideas with some of my peers in my game design class.
 
 
 ### Domain
 _Describe the project's domain in five words._
+Two-player, abstract strategy board games
 
 
 ### Interface (syntax)
 _How might the user interact with the language? What does programming look 
 like? Why is this the right way to interact with the problem domain?_ 
+The user interacts with the language by swapping their colored pieces with
+other pieces on the board. They observe the entire board at each turn, and 
+they try to pick the best move by reasoning through the outcomes of different
+swaps. Involving the whole board throughout the entire game is an intentional
+choice that forces the user to be thinking about many different moves each
+turn. The language environment is complex, even though the vocabulary of the
```

Comment:
I'm curious why you said "the different positions on the board as the governing syntax". To me, the syntax is simple -- you check if there's a piece in a spot you can legally move to, and if there is, you swap the pieces. I'm not quite understanding how the different positions themselves are the syntax.